client/src/components/table.jsx

```diff
@@ -0,0 +1,79 @@
+const React = require("react");
+import PropTypes from "prop-types";
+
+const Table = props => {
+  console.log("props from Table", props);
+  return (
+    <table>
+      <thead></thead>
+      <tbody>
+        <tr>
+          <td>
+            CEO<br></br>
+            {props.about.CEO}
+          </td>
+          <td>
+            Employees<br></br>
+            {props.about.employees}
+          </td>
+          <td>
+            Headquarters<br></br>
+            {props.about.headquarters}
+          </td>
+          <td>
+            Founded<br></br>
+            {props.about.founded}
```

Comment:
Since this .about is an object consider giving a label and value attributes and iterate over the object and populate your table.

Reply:
code has been updated, this no longer applies